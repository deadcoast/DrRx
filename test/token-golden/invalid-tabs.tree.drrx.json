[
  {
    "line": 1,
    "raw": "\t.                        # EXPECTED ERROR: SP.01 tabs disallowed; root also must be column 1 (SP.06)",
    "tokens": [
      {
        "kind": "invalid.tab",
        "start": 0,
        "end": 1,
        "text": "\t"
      },
      {
        "kind": "root",
        "start": 1,
        "end": 2,
        "text": "."
      },
      {
        "kind": "comment",
        "start": 26,
        "end": 101,
        "text": "# EXPECTED ERROR: SP.01 tabs disallowed; root also must be column 1 (SP.06)"
      }
    ]
  },
  {
    "line": 2,
    "raw": "",
    "tokens": []
  }
]