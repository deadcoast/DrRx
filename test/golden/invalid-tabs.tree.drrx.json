[
  { "rule": "SP.01", "line": 1, "column": 1 },
  { "rule": "SP.06", "line": 1, "column": 1 }
]
